qiskit/ignis/characterization/characterization_utils.py

```diff
@@ -39,8 +39,6 @@ def pad_id_gates(circuit, qr, qubit, num_of_id_gates):
         # the case of .iden should be removed once terra 0.13 is stable
         if hasattr(circuit, 'i'):
```

Comment:
Can you remove this `if` statement and the comment too. We have the minimum terra version set to 0.13.0 already so we don't need to need to chexk if `i()` is available anymore.

Reply:
Done I have removed the IF statement and the comments